test/commands/deploy/pipeline.test.ts

```diff
@@ -117,6 +125,10 @@ describe('deploy pipeline', () => {
     test
       .stdout()
       .stderr()
+      .do(() => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        executeCommandStub = sandbox.stub(PromoteCommand.prototype, 'executePromotion' as any);
```

Comment:
Why do we need this?

Reply:
I use it to mock the executePromotion() method from the base class. Since this is a unit test I want to verify the behaviour of _deploy pipeline_ command only and not the behaviour of the base class. I just assert that the method wass called.

Reply:
:nice: